src/newfight.cpp

```diff
@@ -1115,6 +1115,7 @@ void hit_with_multiweapon_toggle(struct char_data *attacker, struct char_data *v
   int bod_success = 0;
   int bod = GET_BOD(def->ch) + (def->too_tall ? 0 : GET_BODY(def->ch));
   int bod_dice = 0;
+  UNUSED(bod_dice);
```

Comment:
Is bod_dice actually used in calculations? These unused-variable warnings are usually pretty reliable from what I've seen.

Reply:
Hmm you're right. I saw the awake check and the spirit defense part and I wrongly assumed it's used heh. It's actually not. Remove it? Or use it?

Reply:
I also need to go back and recheck the rest I have tagged. Some of them were used in ifdefs (which was why I added the macro in the first place as there's no way around that) but I may have hastily tagged a few that I saw being set without checking if they're actually consumed.

Reply:
I don't know what it was for in the first place, but if it's not being used now then it's probably not needed. Might be worth running the function's logic by a SR3 expert to make sure we're not missing something here, I definitely don't qualify there since everything I know about the system I learned from the codebase.

Reply:
From what I see it is meant to be used for 2 purposes. To eliminate your target's body cpool dice (soaking) in case they're sleeping and to allow a player character to use their skill level in conjuring as cpool body dice. This was meant to make the dice 0 in these cases but frankly we can implement both of these cases without using an extra variable at all. So I'll remove it and ask if someone can tell me about the sleeping and conjuring rules and add those directly if they actually apply.